### DISCUSSION TITLE
as_shape

### DISCUSSION THREAD

**Discussion by prb000** [cleder/pygeoif#205] [General]
it seems there's a discrepancy between the tle2czml library's expectations and the actual pygeoif library's contents. Specifically, the tle2czml code is attempting to import and use an as_shape function from pygeoif.geometry, which does not exist in the pygeoif version I've downloaded in version 1.2.0?   Any ideas

**Comment by cleder** [comment]
czml has not been updated in a long while and is unmaintained. you can try and version peg pygeoif to version 0.x or fork czml and update it for 1.x
